useshortcut/shortcut-client-js
# Update Swagger definitions, deps, and library components

- Update to latest Swagger definition
- Update README.md with new organization namespace and documentation fixes
- Add `format` script to format via Prettier
- Update all dependencies to latest (This should resolve all security vulnerabilities)
- Ensure CI uses `small` resources
- Migrate the transform function into TypeScript
- Prepare for new 1.1.0 release.
- Update primary logo

### Todo
- [ ] Verify and publish docs
- [ ] Publish `@shortcut/client` - 1.1.0
- [ ] Publish `@useshortcut/client` - 1.1.0
- [ ] (Optionally) Deprecate the legacy @clubhouse.io NPM package.
